scripts/install.js

```diff
@@ -109,18 +110,63 @@ function checkAndDownloadBinary() {
       return;
     }
 
-    download(sass.getBinaryUrl(), sass.getBinaryPath(), function(err) {
-      if (err) {
-        console.error(err);
-        return;
-      }
+    var tmpPath = getTempPath();
+    if (!(sass.hasBinary(tmpPath))) { // download and install
+      download(sass.getBinaryUrl(), tmpPath, function(err) {
+        if (err) {
+          console.error(err);
+          return;
+        }
+        console.log('Binary downloaded at', tmpPath);
+        //copy only, if not already in vendor
+        if (tmpPath !== sass.getBinaryPath()) {
+          copyBinary(tmpPath);
+        }
+      });
 
-      console.log('Binary downloaded and installed at', sass.getBinaryPath());
-    });
+    } else { // install only
+      copyBinary(tmpPath);
+    }
   });
 }
 
 /**
+ * Find a temp folder for file
+ *
+ * @returns {string} temp folder including binary file name
+ * @api private
+ */
+
+function getTempPath() {
+  var candidateTmpDir = tmpdir() || process.env.npm_config_tmp;
```

Comment:
Please take a look at the whole function. I removed the unnecessary foreach() loop of this list. It's now either `tmpdir()`, or `process.env.npm_config_tmp`. If they fail, `sass.getBinaryPath()` is used as fallback.

Maybe we should (temporary) disable the skip_ci part of this install script to test it on your CI.